testdata/baselines/reference/astnav/GetTouchingPropertyName.mapCode.ts.baseline.txt

```diff
@@ -1,3 +1,39 @@
+〚Positions: [729, 730]〛
+【TS: SourceFile [0, 9772)】
+《Go: EndOfFileToken [9770, 9772)》
```

Comment:
@andrewbranch I don't know how to read these baselines, but this could be the difference in GetTokenAtPosition that I was worried about.